NRasterizer.CLI/Program.cs

```diff
@@ -27,12 +27,17 @@ private static void BlitTo(Raster raster, Bitmap bitmap)
             {
                 bitmapData = b.LockBits(Entire(b), ImageLockMode.WriteOnly, b.PixelFormat);
                 int si = 0;
-                IntPtr di = bitmapData.Scan0;
-                for (int y = 0; y < b.Height; y++) 
+
+                unsafe
                 {
-                    Marshal.Copy(raster.Pixels, si, di, b.Width);
-                    si += raster.Stride;
-                    di += bitmapData.Stride;
+                    IntPtr di = bitmapData.Scan0;
+                    byte* di1 = (byte*)di;
+                    for (int y = 0; y < b.Height; y++)
+                    {
+                        Marshal.Copy(raster.Pixels, si, (IntPtr)di1, b.Width);
+                        si += raster.Stride;
+                        di1 += bitmapData.Stride; 
+                    }
```

Comment:
The fixed the err on .NET20,
![err_on_net20](https://cloud.githubusercontent.com/assets/7447159/19410031/debc4f84-930b-11e6-8d6a-7c79f062dd92.png)

but not found on .net >=4

----
but is ok for test project to run on .net >=4 
so I will remove this.